### PR TITLE
417 async assets

### PR DESCRIPTION
**Pull request by herpaderpaldent**
This is a prerequisite to: https://github.com/eveseat/web/pull/209

### CODE REVIEW

File: src/Repositories/Character/Assets.php [hunk 1]
```diff
@@ -34,16 +34,19 @@
...
+     * @return \Illuminate\Database\Eloquent\Builder
      */
-    public function getCharacterAssets(int $character_id): Collection
+    public function getCharacterAssetsBuilder(Collection $character_ids): Builder
     {
 
         return CharacterAsset::with('content', 'type')
+            ->leftJoin('invTypes', 'character_assets.type_id', '=', 'invTypes.typeID')
             ->select(DB::raw('
                 *, CASE
```
Comment (warlof) [inline src/Repositories/Character/Assets.php]: Surrounding the `CASE` with `IFNULL` seems to provide proper results.
```
*, IFNULL(CASE.......END, "Unknown Location")
```

Comment (herpaderpaldent) [reply src/Repositories/Character/Assets.php]: my latest commit should address asset-safety in hibernation: https://github.com/eveseat/services/pull/66/commits/9216f31f14a1df35fd92a54ca64da3d8e0e85c86

**Comment by herpaderpaldent** [comment]
I certainly will do corp-assets too. Current Priority Backlog is this:
1. character assets-view (/) * with this PR
2. configuration/users 
3. various datatables issues on seat including corp-assets

**Comment by herpaderpaldent** [comment]
can this be merged?

**Pull request merged**